<!doctype html>
<html>
<head>
  <meta charset=utf-8 />
  <link href="media/vish-screen.css" type="text/css" rel="stylesheet" media="screen">
  <link href="media/vish-print.css" type="text/css" rel="stylesheet" media="print">
</head>
<body>
  <div id="main">
    <div id="content">
<h1 id="viswanadham-sridhara">Viswanadham (Vish) Sridhara</h1>
<h2 id="contact">CONTACT</h2>
<p>firstname.lastname@gmail.com &emsp;  (301) 857 2931 &emsp;  US Citizen<br />
<strong>Links: </strong><a href="https://orcid.org/0000-0003-0688-6140">ORCID</a> | <a href="https://www.linkedin.com/in/viswanadham-sridhara-308263174">LinkedIn</a> | <a href="https://bitbucket.org/viswam78/">BitBucket</a> | <a href="https://sridhara-omics.github.io/">Blog</a> | <a href="https://github.com/sridhara-omics">Github</a> </p>
<h2 id="profile">PROFILE</h2>
<p>[1] Experienced bioinformatics scientist / data engineer with <strong>12+ years experience</strong> in building bioinformatics pipelines for Next Generation Sequencing (NGS) and mass spectrometry data.<br />
[2] Track record of producing high-quality publications (<strong>20+ journal articles, google scholar citations:1000+, h-index=15</strong>) including data science and ML methodologies.<br />
[3] Proven ability to work effectively in both small and large groups, utilizing agile methodologies such as Scrum, Kanban to drive collaboration and deliver results effectively.

</p>
<h2 id="education">EDUCATION / CERTIFICATIONS</h2>
<p>PHD '07, Electrical and Computer Engg (<strong>Bioinformatics</strong>), Old Dominion University, USA<br />
<p>MS '03, Electrical and Computer Engg, Old Dominion University, Norfolk, VA, USA<br />
<p>• Deep learning specialization (Deeplearning.ai)<br />
• Microsoft Azure Databricks for Data Engineering (Microsoft)<br />
• Foundations of Project Management (Google)
</p>
<h2 id="technical-skills">SKILLS</h2>
<p><strong>Programming: </strong>Python, R, Nextflow, MySQL, Jira, Rally, AWS, Git, Databricks, HPC<br />
<p><strong>NGS/Mass-spectrometry: </strong>Bulk/sc RNA-seq, Proteomics, Protein language models (AI/ML)<br />
<p><strong>Soft-skills: </strong>Strong problem solving skills, Document highly technical applications, Teamwork<br />
<p><strong>Bioinformatics: </strong>Antibody (IMGT, IgBlast, OAS), Genomics (TCGA), Proteomics (CPTAC)<br />
<h2 id="professional-experience">PROFESSIONAL EXPERIENCE</h2>
<h3 id="jul-22---present">Jul ’22 - present </h3>
<p><em>Senior Bioinformatics Scientist</em><br />
<strong>Argentys, Remote role</strong></p>
<p>• Developed and maintained a highly efficient and scalable antibody design pipeline using Observed Antibody Space sequence data (3TB data, 2 billion sequences of single cell BCR).<br /> 
• Leveraged the Immcantation suite of tools, IgBlast, IgPhyMl programs, to build a focused lineage dataset of 5 million parent-child pairs with evolutionary information.<br />
• Built a Python package <strong>OAS-wrapper</strong> (available on PyPI) for antibody sequence analysis.<br />
• Built a R module <strong>scPipeline</strong>, for integrating essential single cell Analysis tools.</p>
<h3 id="apr-21---jul-22">Apr ’21 - Jul ’22</h3>
<p><em>Software Engineer - Test and Technical Writer</em><br />
<strong>Protein Metrics, a Dotmatics company, Remote role</strong></p>
<p>• Developed and implemented automated command-line interface (CLI) tests for Byos applications, with a focus on antibody analysis (de novo, reduced, oxidation, deamidation).<br />
• Regularly updated 15+ customer-facing product manuals on a quarterly schedule, enabling advanced users to levarage biotherapeutic characterization workflows.<br />
• Integrated FDR sensitivity calculations into Byonic CI/CD pipeline for automated version testing for every jenkins build.
</p>
<h3 id="jun-19---apr-21">Jan ’19 - Apr '21</h3>
<p><em>Senior Bioinformatics Scientist</em><br />
<strong>Food and Drug Administration (Argentys contractor), Silver Spring, MD</strong></p>
<p>• Built an end-to-end RNA-seq pipeline using fastp, HISAT2, featureCounts, and DESEQ2. The pipelline is being used by internal FDA reviewers and researchers on a daily basis.<br />
• Developed a single-cell RNA-seq pipeline for 10X datasets using Cell Ranger, SAVER, Seurat/Monocle, fastMNN, and R packages.<br />
• Characterized immune-driven (CD4/CD8 T cells) drug hypersensitivity using single-cell RNA sequencing technologies (10X Genomics).</p>
<p><em>Freelance Bioinformatician</em><br />
• At Mass General Hospital, improved primer design to capture long genomic regions.<br />
<h3 id="jun-16---nov-18">Jun ’16 - Nov ’18</h3>
<p><em>Bioinformatics Systems Developer</em><br />
<strong>Booz Allen Hamilton, Rockville, MD</strong></p>
<p>• Participated in design of cloud-based tools using EC2, S3, and Lambda (AWS technologies), to create scalable and cost-effective solutions for data storage, processing, and analysis.<br />
• Developed a genomics pipeline using Snakemake, R packages, and Python modules for PAT, a cloud-based placental atlas tool that uses raw data from the NCBI GEO database.<br />
• Improved analytics for differential expression, gene ontology and pathway analysis for the PAT tool (<strong>>12500 datasets for >20 RNA-seq and >300 microarray projects</strong>).</p>
<p><em>Freelance Bioinformatician</em><br />
• At biofuel startup, set-up in-house Jbrowse genome browser and BLAST+ server.<br />
<h3 id="dec-12---apr-16">Jan ’12 - Apr ’16</h3>
<p><em>Research Engineering/Scientist Associate V</em> Jan ’12 - Apr ’16<br />
<strong>The University of Texas at Austin – CENTER FOR COMPUTATIONAL BIOLOGY AND BIOINFORMATICS</strong></p>
<p>
• Army research office MURI grant (multi university, incl. Boston University) to build mathematical models to infer associations in multi-omics data <br />
  a. Novel PTM identification using Unrestricted bioinformatic search of large-scale mass-spectrometry based proteomics data generated for mutiple E. coli growth conditions. <br />
  b. 13C Metabolic flux analyses for the same above mentioned samples using FiatFlux.<br />
• Built machine learning models using LASSO regression to predict bacterial growth conditions from <em>in-silico</em> metabolic fluxes generated from metabolic network based flux balance analyses (FBA) methods.<br />
• Helped multiple labs with proteomics bioinformatics consulting and short-term contracts.<br />
• Taught computational mass-spectrometry in Big Data Summer School organized by CCBB (Center for Computational Biology and Bioinformatics every summer – 4 day course)
<br>
• Bioinformatics proteomics consulting work with UT Southwestern investigators while Staff at UT Southwestern Medical Center.</p>

<h3 id="jun-07---jan-12">Jun ’07 - Jan ’12</h3>
<p><em>Postdoctoral Fellow</em><br />
  <strong>National Center for Biotechnology Information (NCBI)</strong></p>
  <p>• Improved signal processing routines/filters and optimized OMSSA search algorithm to accommodate high-resolution mass-spectrometry data.<br />
  • Global customer success support providing sensitive analysis to advanced users.<br />
  • Developed a high-throughput method to integrate phosphoproteomics and sequence data for automatic annotation of funcitonal sites using NCBI toolkit, RefSeq and CDD.<br />
  
<h3 id="aug-03---may-07">Aug ’03 - May ’07</h3>
<p><strong>OLD DOMINION UNIVERSITY (SIMULATION LAB, FRANK REIDY RESEARCH CENTER FOR BIOELECTRICS)</strong></p>
<p><em>Graduate Research Assistant</em> May ’02 - May ’07<br />
<strong>COMPUTATIONAL RESEARCH LAB - BIOELECTRICS</strong></p>
<p>• DoD MURI grant (multi university, incl. MIT) to model/simulate how nano-second pulsed electric fields (nsPEFs) kill cancer cells<br />
• Compute intense Molecular Dynamics (MD) simulations to understand multiple nanopore formation in lipid bilayers using advanced high performance computing environment.<br />
• Microscopic calculation of local lipid membrane permittivities and diffusion coefficients.</p>
<p>
<h2 id="awards">AWARDS</h2>
<p>2007: NIH Postdoctoral Fellowship<br />
2007: Outstanding Graduate Researcher Award, Old Dominion University<br />
2006: BCET Scholarship, Old Dominion University<br />
</p>  
<h2 id="professional memberships">MEMBERSHIPS</h2>
<p>American Association for Cancer Research, American society for mass spectrometry</p>
<h2 id="professional memberships">VOLUNTEER</h2>
<p>Chess workshops, NSF Grant Reviewer, JPR / PLoS Pathogens Journal Reviewer</p>
<h2 id="publications">PUBLICATIONS</h2><br />
<ol type="1">
<p><strong> Data Science / Genomics / Proteomics Machine Learning (Selected 10 out of 23)</strong></p>
<li><p>Tosi, L., Sridhara, V., et. al., Nature Biomedical Engineering, 2017.</p></li>
<li><p>Brown, C. W., Sridhara, V., et. al., BMC Genomics 2017.</p></li>
<li><p>Kalantari, R., et. al., RNA 2016.</p></li>
<li><p>Houser, J. R., et. al., PLOS Computational Biology 2015.</p></li>
<li><p>Sridhara, V., et. al., PLoS One 2014.</p></li>
<li><p>Sridhara, V., Joshi, R. P. Biochimica Biophysica Acta - Biomembranes 2014.</p></li>
<li><p>Sridhara, V., et. al., Proteome Science 2012.</p></li>
<li><p>Sridhara, V., et. al., Database (Oxford) 2011.</p></li>
<li><p>Hu, Q., et. al., (invited paper) IEEE Transactions on Plasma Science 2006.</p></li>
<li><p>Joshi, R. P., Sridhara, V., et. al., Journal of Applied Physics 2003.</p></li>
</ol>
