---
layout: archive
title: "CV"
permalink: /cv/
author_profile: true
redirect_from:
  - /resume
---

{% include base_path %}

  

Viswanadham (Vish) Sridhara
===========================

CONTACT
-------

firstname.lastname@gmail.com   US Citizen  
**Links:** [ORCID](https://orcid.org/0000-0003-0688-6140) | [LinkedIn](https://www.linkedin.com/in/viswanadham-sridhara-308263174) | [BitBucket](https://bitbucket.org/viswam78/) | [Blog](https://sridhara-omics.github.io/) | [Github](https://github.com/sridhara-omics)

PROFILE
-------

\[1\] Experienced bioinformatics scientist / data engineer with **12+ years experience** in building bioinformatics pipelines for Next Generation Sequencing (NGS) and mass spectrometry data.  
\[2\] Track record of producing high-quality publications (**20+ journal articles, google scholar citations:1000+, h-index=15**) including data science and ML methodologies.  
\[3\] Proven ability to work effectively in both small and large groups, utilizing agile methodologies such as Scrum, Kanban to drive collaboration and deliver results effectively.

EDUCATION / CERTIFICATIONS
--------------------------

PHD '07, Electrical and Computer Engg (**Bioinformatics**), Old Dominion University, USA  

MS '03, Electrical and Computer Engg, Old Dominion University, Norfolk, VA, USA  

• Deep learning specialization (Deeplearning.ai)  
• Microsoft Azure Databricks for Data Engineering (Microsoft)  
• Foundations of Project Management (Google)

SKILLS
------

**Programming:** Python, R, Nextflow, MySQL, Jira, Rally, AWS, Git, Databricks, HPC  

**NGS/Mass-spectrometry:** Bulk/sc RNA-seq, Proteomics, Protein language models (AI/ML)  

**Soft-skills:** Strong problem solving skills, Document highly technical applications, Teamwork  

**Bioinformatics:** Antibody (IMGT, IgBlast, OAS), Genomics (TCGA), Proteomics (CPTAC)  

PROFESSIONAL EXPERIENCE
-----------------------

### Jul ’22 - present

_Senior Bioinformatics Scientist_  
**Argentys, Remote role**

• Developed and maintained a highly efficient and scalable antibody design pipeline using Observed Antibody Space sequence data (3TB data, 2 billion sequences of single cell BCR).  
• Leveraged the Immcantation suite of tools, IgBlast, IgPhyMl programs, to build a focused lineage dataset of 5 million parent-child pairs with evolutionary information.  
• Built a Python package **OAS-wrapper** (available on PyPI) for antibody sequence analysis.  
• Built a R module **scPipeline**, for integrating essential single cell Analysis tools.

### Apr ’21 - Jul ’22

_Software Engineer - Test and Technical Writer_  
**Protein Metrics, a Dotmatics company, Remote role**

• Developed and implemented automated command-line interface (CLI) tests for Byos applications, with a focus on antibody analysis (de novo, reduced, oxidation, deamidation).  
• Regularly updated 15+ customer-facing product manuals on a quarterly schedule, enabling advanced users to levarage biotherapeutic characterization workflows.  
• Integrated FDR sensitivity calculations into Byonic CI/CD pipeline for automated version testing for every jenkins build.

### Jan ’19 - Apr '21

_Senior Bioinformatics Scientist_  
**Food and Drug Administration (Argentys contractor), Silver Spring, MD**

• Built an end-to-end RNA-seq pipeline using fastp, HISAT2, featureCounts, and DESEQ2. The pipelline is being used by internal FDA reviewers and researchers on a daily basis.  
• Developed a single-cell RNA-seq pipeline for 10X datasets using Cell Ranger, SAVER, Seurat/Monocle, fastMNN, and R packages.  
• Characterized immune-driven (CD4/CD8 T cells) drug hypersensitivity using single-cell RNA sequencing technologies (10X Genomics).

_Freelance Bioinformatician_  
• At Mass General Hospital, improved primer design to capture long genomic regions.  

### Jun ’16 - Nov ’18

_Bioinformatics Systems Developer_  
**Booz Allen Hamilton, Rockville, MD**

• Participated in design of cloud-based tools using EC2, S3, and Lambda (AWS technologies), to create scalable and cost-effective solutions for data storage, processing, and analysis.  
• Developed a genomics pipeline using Snakemake, R packages, and Python modules for PAT, a cloud-based placental atlas tool that uses raw data from the NCBI GEO database.  
• Improved analytics for differential expression, gene ontology and pathway analysis for the PAT tool (**\>12500 datasets for >20 RNA-seq and >300 microarray projects**).

_Freelance Bioinformatician_  
• At biofuel startup, set-up in-house Jbrowse genome browser and BLAST+ server.  

### Jan ’12 - Apr ’16

_Research Engineering/Scientist Associate V_ Jan ’12 - Apr ’16  
**The University of Texas at Austin – CENTER FOR COMPUTATIONAL BIOLOGY AND BIOINFORMATICS**

• Army research office MURI grant (multi university, incl. Boston University) to build mathematical models to infer associations in multi-omics data  
  a. Novel PTM identification using Unrestricted bioinformatic search of large-scale mass-spectrometry based proteomics data generated for mutiple E. coli growth conditions.  
  b. 13C Metabolic flux analyses for the same above mentioned samples using FiatFlux.  
• Built machine learning models using LASSO regression to predict bacterial growth conditions from _in-silico_ metabolic fluxes generated from metabolic network based flux balance analyses (FBA) methods.  
• Helped multiple labs with proteomics bioinformatics consulting and short-term contracts.  
• Taught computational mass-spectrometry in Big Data Summer School organized by CCBB (Center for Computational Biology and Bioinformatics every summer – 4 day course)  
• Bioinformatics proteomics consulting work with UT Southwestern investigators while Staff at UT Southwestern Medical Center.

### Jun ’07 - Jan ’12

_Postdoctoral Fellow_  
**National Center for Biotechnology Information (NCBI)**

• Improved signal processing routines/filters and optimized OMSSA search algorithm to accommodate high-resolution mass-spectrometry data.  
• Global customer success support providing sensitive analysis to advanced users.  
• Developed a high-throughput method to integrate phosphoproteomics and sequence data for automatic annotation of funcitonal sites using NCBI toolkit, RefSeq and CDD.  

### Aug ’03 - May ’07

**OLD DOMINION UNIVERSITY (SIMULATION LAB, FRANK REIDY RESEARCH CENTER FOR BIOELECTRICS)**

_Graduate Research Assistant_ May ’02 - May ’07  
**COMPUTATIONAL RESEARCH LAB - BIOELECTRICS**

• DoD MURI grant (multi university, incl. MIT) to model/simulate how nano-second pulsed electric fields (nsPEFs) kill cancer cells  
• Compute intense Molecular Dynamics (MD) simulations to understand multiple nanopore formation in lipid bilayers using advanced high performance computing environment.  
• Microscopic calculation of local lipid membrane permittivities and diffusion coefficients.

AWARDS
------

2007: NIH Postdoctoral Fellowship  
2007: Outstanding Graduate Researcher Award, Old Dominion University  
2006: BCET Scholarship, Old Dominion University  

MEMBERSHIPS
-----------

American Association for Cancer Research, American society for mass spectrometry

VOLUNTEER
---------

Chess workshops, NSF Grant Reviewer, JPR / PLoS Pathogens Journal Reviewer

PUBLICATIONS
------------

  

**Data Science / Genomics / Proteomics Machine Learning (Selected 10 out of 23)**

2.  Tosi, L., Sridhara, V., et. al., Nature Biomedical Engineering, 2017.
    
3.  Brown, C. W., Sridhara, V., et. al., BMC Genomics 2017.
    
4.  Kalantari, R., et. al., RNA 2016.
    
5.  Houser, J. R., et. al., PLOS Computational Biology 2015.
    
6.  Sridhara, V., et. al., PLoS One 2014.
    
7.  Sridhara, V., Joshi, R. P. Biochimica Biophysica Acta - Biomembranes 2014.
    
8.  Sridhara, V., et. al., Proteome Science 2012.
    
9.  Sridhara, V., et. al., Database (Oxford) 2011.
    
10.  Hu, Q., et. al., (invited paper) IEEE Transactions on Plasma Science 2006.
    
11.  Joshi, R. P., Sridhara, V., et. al., Journal of Applied Physics 2003.