<!doctype html>
<html>
<head>
  <meta charset=utf-8 />
  <link href="media/vish-screen.css" type="text/css" rel="stylesheet" media="screen">
  <link href="media/vish-print.css" type="text/css" rel="stylesheet" media="print">
</head>
<body>
  <div id="main">
    <div id="content">
<h1 id="viswanadham-sridhara">Viswanadham (Vish) Sridhara</h1>
<h2 id="contact">CONTACT</h2>
<p><strong>Contact: </strong>Sridhara.Omics@gmail.com &emsp;  US Citizen<br />
<strong>Links: </strong><a href="https://scholar.google.com/citations?hl=en&user=6rYid2YAAAAJ&view_op=list_works&sortby=pubdate">Google Scholar</a> | <a href="https://www.linkedin.com/in/viswanadham-sridhara-308263174">LinkedIn</a> | <a href="https://github.com/sridhara-omics">Github</a> | <a href="https://bitbucket.org/viswam78/">BitBucket</a> | <a href="https://sridhara-omics.github.io/">Blog</a> | <a href="https://hub.docker.com/u/sridharabio">DockerHub</a></p>
<h2 id="profile">PROFILE</h2>
<p>[1] Experienced bioinformatics data scientist with over <strong>12 years of experience</strong> in nextgeneration sequencing (NGS) and mass-spectrometry software applications.<br />
[2] Skilled in using machine learning and deep learning methods for high-dimensional biological datasets, including <strong>proteogenomics, multi-omics and single-cell applications</strong>.<br />
[3] Adept in utilizing high-performance computing methods and technologies such as <strong>snakemake, docker containerization, and cloud computing</strong>.<br />
[4] Proven ability to work effectively in both small and large groups, utilizing <strong>agile project management methodologies</strong>.<br />
[5] Track record of producing high-quality results with <strong>~20 peer-reviewed publications</strong> (google scholar citations: 1000+, h-index=15, incl. Nature Biomedical Engineering IF=27.0).
</p>
<h2 id="education">EDUCATION</h2>
<p><strong>2003 - 2007: PHD</strong>, Electrical and Computer Engineering, ODU, Norfolk, VA<br />
<strong>Dissertation (funded by Department of Defense MURI grant):</strong> Molecular dynamics simulations to probe effects of ultra-short high-intensity electric pulses on biological suspensions to understand how nano-second pulsed electric fields (nsPEFs) kill cancer cells.</p>
<p><strong>2001 - 2003: MS</strong>, Electrical and Computer Engineering, ODU, Norfolk, VA<br />
<strong>Thesis:</strong> Monte carlo analysis of millimeter-wave GaN Gunn diodes.</p>
<h2 id="certifications">PROFESSIONAL CERTIFICATES</h2>
<p>[1] Microsoft Azure Databricks for Data Engineering (Microsoft)<br />
[2] Google Cloud Fundamentals: Core Infrastructure (Google)<br />
[3] Foundations of Project Management (Google)
</p>
<h2 id="technical-skills">TECHNICAL SKILLS</h2>
<p><strong>Programming: </strong>Python, R, SQL, MATLAB, PyTorch, Databricks (PySpark, SparkSQL), Shell Scripting<br />
<p><strong>Machine learning: </strong>Regression, classification, clustering, deep learning frameworks<br />
<p><strong>NGS: </strong>Bulk RNA-seq, Single cell RNA-seq, Long read sequencing, DNA-seq/WGS<br />
<p><strong>Mass spectrometry: </strong>Identification, Quantitation, PTMs, PPIs, DDA, DIA<br />
<p><strong>Structure and SysBio: </strong>AlphaFold2, MetaCyc, Immcantation, GROMACS, VMD<br />
<p><strong>Pipelines / Data Portals: </strong>Snakemake, Nextflow, ICGC, COSMIC, Tranche<br />
<p><strong>DevOps / MLOps: </strong>Git, Docker, CI/CD, Databricks (Azure), Sagemaker (AWS)</p>
<h2 id="professional-experience">PROFESSIONAL EXPERIENCE</h2>
<h3 id="jun-23---jun-24">Jun ’23 - Jun ’24</h3>
<p><em>Senior Bioinformatics Scientist</em><br />
<strong>Argentys, Remote role</strong></p>
<p>• Developed a highly efficient antibody design pipeline using evolutionary information from antibody genomic sequence data (single cell BCR).<br /> 
• Leveraged the Immcantation suite of tools, IgBlast, IgPhyMl programs, and the OAS (Observed Antibody Space) database.<br /> 
• Conducted preliminary analysis using Databricks on a substantial dataset of 1.4 million paired heavy-light chain sequences.</p>
<h3 id="jul-22---jun-23">Jul ’22 - Jun ’23</h3>
<p><em>Career break</em><br />
<strong>Travel, online learning and family</strong></p>
<p>• Leveraged online learning platforms and resources to gain new skills and knowledge in relevant areas, including deep learning, cloud computing etc.
</p>
<h3 id="apr-21---jul-22">Apr ’21 - Jul ’22</h3>
<p><em>Software Engineer - Test and Technical Writer</em><br />
<strong>Protein Metrics, a Dotmatics company, Remote role</strong></p>
<p>• Conducted FDR sensitivity calculations for Byonic CI/CD version testing.<br />
• Developed and implemented automated command-line interface (CLI) tests for Byos applications, with a focus on antibody de novo assembly, hydrogen-deuterium exchange (HDX) studies and other proteomics applications.<br />
• Routinely updated customer-facing product manuals (15+) and release notes.</p>
<h3 id="jun-19---apr-21">Jun ’19 - Apr '21</h3>
<p><em>Senior Bioinformatics Scientist</em><br />
<strong>Food and Drug Administration (Argentys contractor), Silver Spring, MD</strong></p>
<p>• Built an RNA-seq pipeline for bulk processing of large-scale genomics datasets using fastp, HISAT2, featureCounts, and DESEQ2.<br />
• Developed a single-cell RNA-seq pipeline prototype for analyzing 10X datasets using Cell Ranger, SAVER, Seurat/Monocle, fastMNN, and R packages.<br />
• Characterized immune-driven (CD4/CD8 T cells) drug hypersensitivity using single-cell RNA sequencing technologies (10X Genomics).<br />
• Identified molecular features of neural stem cells during different stages of differentiation.<br />
• Conducted single-cell RNA-seq/ATAC-seq analyses (10X Genomics) of bone-marrow and adipose-derived MSCs using Cell Ranger, Monocle, CICERO and other related packages.</p>
<h3 id="nov-18---may-19">Nov ’18 - May ’19</h3>
<p><em>Part-time Test Engineer</em><br />
<strong>Protein Metrics, a Dotmatics company, Cupertino, CA</strong></p>
<p>• Benchmarking of different Byos applications and comparative studies with leading proteomics software.<br />
• Advanced intact protein analysis for data coming from different instruments.</p>
<h3 id="feb-18---nov-18">Feb ’18 - Nov ’18</h3>
<p><em>Bioinformatics Systems Developer</em><br />
<strong>Booz Allen Hamilton, Rockville, MD</strong></p>
<p>• Participated in design of cloud-based tools using EC2, S3, and Lambda (AWS technologies), to create scalable and cost-effective solutions for data storage, processing, and analysis.<br />
• Developed a genomics pipeline using Snakemake, R packages, and Python modules for PAT, a cloud-based placental atlas tool that uses raw data from the NCBI GEO database.<br />
• Improved analytics for differential expression, gene ontology and pathway analysis for the PAT tool (>12500 datasets for >20 RNA-seq and >300 microarray projects).</p>
<h3 id="apr-16---dec-17">Apr ’16 - Dec ’17</h3>
<p><em>Independent Contractor</em><br />
<strong>Universities, Hospitals and Startups (Remote)</strong></p>
<p>Independent Consultant / Contractor for MGH, WRI and Mascoma<br />
<strong>Tools used: </strong>Bowtie, samtools, bedtools, BCFtools, LUMPY, GATK (Mutect2, HaplotypeCaller), MUMmer, SPAdes<br />
<strong>Mass General Hospital: </strong>Designed primers/probes for a new assay, built an end-to-end bioinformatics pipeline and developed new methods to assess metrics for a long genomic region capture protocol at Mass General Hospital. Resulted in a co-first author publication in Nature Biomedical Engineering.<br />
<strong>Windber Research Institute: </strong>Developed novel statistical methods to integrate multi-omics data of breast tumors at Windber Research Institute, utilizing a combination of machine learning models, clustering methods, and data from DrugBank.<br />
<strong>Mascoma: </strong>Built custom RNA-seq pipeline, analyzed WGS data, and set-up in-house genome browser and blast+ web server for biofuel production at Mascoma.</p>
<h3 id="dec-12---apr-16">Dec ’12 - Apr ’16</h3>
<p><strong>THE UNIVERSITY OF TEXAS SYSTEM (UT AUSTIN, UTSW MEDICAL CENTER DALLAS)</strong></p>
<p><em>Research Engineering/Scientist Associate V</em> Dec ’12 - Aug ’15<br />
<strong>The University of Texas at Austin – CENTER FOR COMPUTATIONAL BIOLOGY AND BIOINFORMATICS</strong></p>
<p>
• Worked on a multi-university Army research office MURI grant to build mathematical models for E. coli multi-omics data inference (34 growth conditions, 9 time points).<br />
• Developed a computational method for unbiased identification and characterization of new post-translational modifications using MODa, Python and R scripts.<br />
• Conducted 13C metabolic flux analyses for E. coli samples using FiatFlux software.<br />
• Developed and optimized machine learning models, including LASSO (elastic net) models, to predict bacterial growth conditions based on in-silico metabolic fluxes (FBA).<br />
• Provided proteomics bioinformatics consulting and taught computational mass-spectrometry in a 4-day course at the Big Data Summer School organized by CCBB.<br />
• Managed and oversaw the development of computational infrastructure, including highperformance computing clusters, storage systems, and software tools, ensuring optimal performance and availability for research and development activities.</p>
<br>
<p><em>Computational Biologist</em> Aug ’15 - Apr ’16<br />
<strong>UT Southwestern Medical Center – PROTEOMICS CORE FACILITY</strong></p>
<p>• Conducted protein identification, quantitation, PTM and protein-protein interaction analyses using FAIR data principles with different bioinformatics tools e.g., OMSSA, MASCOT, Sequest, X!Tandem, ProteomeDiscoverer, MaxQuant, SAINT.<br />
• Provided expert bioinformatics and proteomics consulting for UT Southwestern investigators (in compliance with CAP/CLIA regulations), specializing in the analysis of data related to immunology, oncology, and neurological disorders.</p>

<h3 id="jan-12---dec-12">Jan ’12 - Dec ’12</h3>
<p><em>Scientific Programmer</em><br />
<strong>Samuel Roberts Noble Foundation</strong></p>
<p>• Assisted in integrating GenoLogics LIMS system with mass-spectrometry equipment.<br />
• Built pathway databases using MetaCyc and PathwayTools.</p>
<h3 id="jun-07---jan-12">Jun ’07 - Jan ’12</h3>
<p><em>Postdoctoral Fellow</em><br />
<strong>National Center for Biotechnology Information (NCBI)</strong></p>
<p>• Improved noise filtering routines and optimized data analysis algorithms for OMSSA (open source mass-spectrometry search engine), utilizing ML models such as LDA to achieve robust and accurate identification and quantification of target peptides.<br />
• Provided OMSSA user support for quantitative proteomics techniques (iTRAQ, TMT 6-plex) by adding requested enzymes, modifications and ion-types.<br />
• Developed a new method for annotating post-translational modifications (PTMs) on genomes using high-throughput proteomics data, protein family domain definitions, and NCBI resources. This method enables large-scale annotation of conserved PTM sites and identification of novel evolutionarily conserved PTMs.<br />
• Developed a prototype to overcome limitations in present-generation sequence library search algorithms, such as exact precursor mass match and unknown PTMs (ASMS oral 2011).</p>
<p>
<h3 id="aug-03---may-07">Aug ’03 - May ’07</h3>
<p><strong>OLD DOMINION UNIVERSITY (JOSHI LAB, FRANK REIDY RESEARCH CENTER FOR BIOELECTRICS)</strong></p>
<p><em>Graduate Research Assistant</em> Aug ’03 - May ’07<br />
<strong>COMPUTATIONAL RESEARCH LAB - BIOELECTRICS</strong></p>
<p>• Conducted research under DoD MURI grant (multi university, incl. MIT) to model and simulate the effect of nano-second pulsed electric fields (nsPEFs) on cancer cells.<br />
• Conducted intense Molecular Dynamics simulations to understand nanopore formation in lipid bilayers using GROMACS (analysis) and VMD (visualization).<br />
• Mentored and trained graduate students, providing guidance and support for research projects and career development.</p>
<p>
<p><em>Research Assistant V</em> May ’04 - Dec ’06<br />
<strong>FRANK REIDY RESEARCH CENTER FOR BIOELECTRICS</strong></p>
<p>• Conducted Time Domain Dielectric Transient Spectroscopy (TDDS) measurements to monitor structural changes in leukemic cells after nsPEF application.<br />
• Analyzed TDDS data with MATLAB signal processing toolbox.</p>
<h2 id="selected-talks">SELECTED TALKS</h2>
<ol type="1">
<li><p>Large-scale analysis of Post-Translational Modifications</p></li>
<li><p>Using subspectral interval matches to make novel identifications of ETD tandem mass spectra</p></li>
<li><p>Predicting bacterial growth conditions from bacterial physiology, UT-Austin</p></li>
<li><p>Engineered DNA Probes for Massively Multiplexed Cloning of Large Genome Regions, FDA.</p></li>
</ol>
<h2 id="publications">PUBLICATIONS</h2>
<ol type="1">
<p><em>Data science / Machine Learning / Bioinformatics papers</em><br />
<li><p>Tosi, L., Sridhara, V., et. al., Nature Biomedical Engineering, 2017.</p></li>
<li><p>Caglar, M. U., et. al., Scientific Reports 2017.</p></li>
<li><p>Brown, C. W., Sridhara, V., et. al., BMC Genomics 2017.</p></li>
<li><p>Kalantari, R., et. al., RNA 2016.</p></li>
<li><p>Houser, J. R., et. al., PLOS Computational Biology 2015.</p></li>
<li><p>Sridhara, V., et. al., PLoS One 2014.</p></li>
<li><p>Sridhara, V., Joshi, R. P. Biochimica Biophysica Acta - Biomembranes 2014.</p></li>
<li><p>Sridhara, V., Joshi, R. P. Biochimica Biophysica Acta - Biomembranes 2014.</p></li>
<li><p>Sridhara, V., et. al., Proteome Science 2012.</p></li>
<li><p>Sridhara, V., et. al., Database (Oxford) 2011.</p></li>
<li><p>Joshi, R.P., et. al., IEEE Trans. Dielectrics & Electrical Insulation 2009.</p></li>
<li><p>Allen L. Garner, et. al., Biochemical Biophysical Research Communications 2007.</p></li>
<li><p>Joshi, R.P., et. al., Physical review. E, Statistical, nonlinear, and soft matter physics 2007.</p></li>
<li><p>Pliquett, U., et. al., Bioelectrochemistry 2006.</p></li>
<li><p>Joshi, R.P., Sridhara, V., et. al., Biochemical Biophysical Research Communications 2006.</p></li>
<li><p>Qin Hu, Sridhara, V., et. al., (invited paper) IEEE Transactions on Plasma Science 2006.</p></li>
<li><p>Hu, Q., Sridhara, V., et. al., Physical Review E - Stat, Nonlinear, and Soft Matter Physics 2005.</p></li>
<li><p>Joshi, R. P., Sridhara, V., et. al., Journal of Applied Physics 2003.</p></li>
<li><p>Joshi, R. P., Sridhara, V., et. al., Journal of Applied Physics 2003.</p></li>
</ol>
<h2 id="conference-proceedings">CONFERENCE PROCEEDINGS</h2>
<ol type="1">
<li><p>Sridhara, V., et. al., (invited paper) Bio-Electromagnetic Society Conference 2006.</p></li>
<li><p>Pliquett, U., et. al., Electromed 2005 Conference.</p></li>
<li><p>Garner, A.L., et. al., Electromed 2005 Conference.</p></li>
</ol>
<h2 id="posters">POSTERS</h2>
<p>SCD(FDA) 2020, AACR 2017, Proceedings of the AACR 2016, EMBL Conference 2014, Synthetic Biology: Engineering, Evolution and Design (SEED), ASMS 2010, ASMS 2009, ASMS 2008</p>
<h2 id="grant reviewer">GRANT REVIEWER</h2>
<p>National Science Foundation (NSF)</p>
<h2 id="journals refereed">JOURNALS REFEREED</h2>
<p>JPR, PLoS Pathogens, Journal of Theoretical Biology, Biology Direct, PLoS Computational Biology, International Journal of Bioinformatics Research and Applications</p>
<h2 id="professional memberships">PROFESSIONAL MEMBERSHIPS</h2>
<p>American Association for Cancer Research, American society for mass spectrometry
</p>
<h2 id="awards">AWARDS</h2>
<p>• 2007: NIH Postdoctoral Fellowship<br />
• 2007: Outstanding Graduate Researcher Award, Old Dominion University<br />
• 2006: BCET Scholarship, Old Dominion University<br />
• 2006: Honorary Mention, ODU Research Foundation Day Poster Presentation<br />
• 1996-2000: National Merit Scholarship, College of Engineering, Osmania University<br />
• 1994-1996: College Scholarship, St. Thomas Samhita Jr. College<br />
• 1994: Certificate of Merit in Mathematics
</p>  
<h2 id="sample-projects">SAMPLE PROJECTS</h2>
<h3 id="ml-regularization">ML-Regularization</h3>
<p><a href="https://www.ncbi.nlm.nih.gov/pubmed/25502413">PLoS One paper - Regularization and metabolic network models [URL: https://www.ncbi.nlm.nih.gov/pubmed/25502413]</a>
<h3 id="cancer-proteogenomics">Cancer-omics</h3> 
<p><a href="https://sivome.github.io/Poster_PDFs/Proteogenomic_AACR_Poster.pdf">AACR poster [URL: https://sivome.github.io/Poster_PDFs/Proteogenomic_AACR_Poster.pdf]</a></p>
<h3 id="single-cell-fda-project">single-cell work</h3> 
<p><a href="https://sivome.github.io/Poster_PDFs/SingleCellMSC_SCD_Poster.pdf">SCD (FDA) poster [URL: https://sivome.github.io/Poster_PDFs/SingleCellMSC_SCD_Poster.pdf]</a></p>
<h3 id="neural-networks-and-single-cell-transcriptomics">Neural networks</h3>
<p><a href="https://sivome.github.io/genomics/2019/11/17/Denoising-Single-Cell-Data-With-Autoencoders.html">Neural Networks blog - 10X single cell data [URL: https://sivome.github.io/genomics/2019/11/17/Denoising-Single-Cell-Data-With-Autoencoders.html]</a> 